recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -440,6 +443,7 @@ def package_info(self):
             self.options.with_elasticsearch
         ):
             self.cpp_info.components[self._http_client_name].requires.append("libcurl::libcurl")
+            self.cpp_info.components[self._http_client_name].requires.append("openssl::openssl")
```

Comment:
Hi @Cogitri! Thanks for the contribution.
Shouldn't this also be added to the `opentelemetry_exporter_etw` component?

Reply:
@czoido could you possibly explain it in more details why should it be added, from your point of view? what's the motivation, what's the benefit, etc?

Reply:
@SSE4 It's because the recipe logic, when you have `with_etw=True` it adds _openssl_ as a requirement if I'm not mistaken.